### PR TITLE
Feat/landingpage

### LINKED ISSUE #1: User Story: As a user, when I visit the homepage, I should see a grid of meme images upon page load so that I can find one that I like.
**User Story**
As a user, when I visit the homepage, I should see a grid of meme images upon page load so that I can find one that I like. 

**Acceptance Criteria**
- **Scenario:** Load page
- **Given:** The user navigates to the home page
- **When:** The user views the page
- **Then:** The user should see a grid of meme images

### LINKED ISSUE #10: User Story: As a user, when I visit the homepage, I see the logo and link to saved memes
**User Story**
As a user, when I visit the homepage, I see the logo and link to saved memes.

**Acceptance Criteria**
- **Scenario:** Load page
- **Given:** The user navigates to the home page
- **When:** The user views the page
- **Then:** The user should see a the Meme Depot logo and a link to Saved Memes

### PR DESCRIPTION
**Pull request by sdtrachsel**
## Describe your changes:
[//]: <> (Which files were changed? What changes were made?)
- Installed React Router V5
- Installed React Router Types
- Implemented Router on landing page for header nav links
- Created meme image data
- Created Image componenent and css files *(see notes below)
- Populated Header componenet & simple css
- Populated Images component & simple css
- Populated Image component & simple css

Added an Image component. We originally talked about it possibily being overkill but after research on implementating multiple buttons. The majority suggested a seperate componenet to follow SRP so the Images component would only be managing the contents of the container and the Image component would only be managing the content of the of the buttons themselves. I can back it out if needed. 

I ended up hosting our images on imgur so the links would be easier to access. I couldn't find any documentation on the best place to have the data so I saved it in the src file. I loaded all 19 images. 

Added typing of React.FC to the components of App, Header, Images, Image, and SavedMemes.  I was receiving sporadic void errors from TS without it. 

**Will need to `npm i`  after pull down**
## Checklist:

- [X] My code follows the style guidelines of this project
- [X] I have performed a self-review of my code
- [X] I have tested the functionality of any new or updated features in the browser, if applicable
- [X] I have added a "Reviewer" to review & merge
- [] If applicable, my code passes all tests

If issue found, describe and refernces user story/projectboard issue

### CODE REVIEW

File: src/assets/meme_depot_logo.png
Comment (mrlobatoman) [inline src/assets/meme_depot_logo.png:1]: This looks so good!

Comment (CodeMeister362) [reply src/assets/meme_depot_logo.png:1]: love it

**Review by CodeMeister362 — APPROVED** [review]
Thanks for adding those images and getting to work on some of the components.

File: src/components/header/Header.tsx [hunk 1]
```diff
@@ -1,16 +1,18 @@
...
 
-type GreetProps = {
-	name: string
+const Header: React.FC = () => {
+  return (
+    <header>
+      <img src={logo} alt="Meme Depot Logo" className="logo" />
+      <nav>
+        <NavLink exact to="/" className={isActive => "nav-link" + (!isActive? ' unselected': "")} > Home </NavLink>
+        <NavLink exact to="/savedmemes" className={isActive => "nav-link" + (!isActive? ' unselected': "")}> Saved Memes </NavLink>
```
Comment (mrlobatoman) [inline src/components/header/Header.tsx:12]: Can you please explain whats happening in the NavLink? More specifically, the className. Thanks!

Comment (sdtrachsel) [reply src/components/header/Header.tsx:12]: @mrlobatoman It is toggling the class of the nav links for styling. 
It's the 'fancy' part of NavLinks.  The active tab right now shows orange with an underline while the inactive shows just black text.

**Pull request merged**